### ISSUE TITLE
[BUG] RHEL-07-040170 clobbers RHEL-07-010040's work

### ISSUE THREAD

**Issue by ferricoxide** [plus3it/ash-linux-formula#275]
**Describe the bug**
RHEL-07-010040 overwrites /etc/issue as it ought ...but then RHEL-07-040170 overwrites _that_. Need to change RHEL-07-040170's `file.managed` to a simple existence-check requisite for the sshd_config.

**Severity**

Prevents newly-added functionality from working as expected

**To Reproduce**
1. Launch new instance
2. Run watchmaker's saltstack in High state
3. Find that `/etc/issue` has the default rather than override/pillar

**Expected behavior**
Pillar content should be in `/etc/issue`

**Additional context**

**Fix Suggestions**

Get rid of `file.managed` block from `RHEL-07-040170` and replace with a file-existence check.

**Work Around

Initiate a standalone run of `RHEL-07-010040` after the Highstate run completes.

**Comment by ferricoxide** [comment]
@ewierschke Feel free to follow this

**Comment by lorengordon** [comment]
In this case, can probably use the salt state `file.touch` as the dependency, or use `file.managed` without any source/content/template options, to avoid excessive jinja if statements.

**Comment by ferricoxide** [comment]
Alright, so `/etc/issue` _should_ always exist in some form or another:
~~~
# rpm -qf /etc/issue
centos-release-7-6.1810.2.el7.centos.x86_64
~~~
So, it likely makes sense to simply remove the:
~~~
file_{{ stig_id }}-{{ parmValu }}:
  file.managed:
    - name: '{{ parmValu }}'
    - source: salt://{{ helperLoc }}/issue.txt
    - user: 'root'
    - group: 'root'
    - mode: 0644
~~~
Block entirely. To be perfectly honest, I can't remember why I had it there in the first place ...because I'm not actually even using a `require` to justify tracking the file by any means.

**Comment by ferricoxide** [comment]
Closed by #276
